DPPP/test/unit/tH5Parm.cc

```diff
@@ -1,40 +1,44 @@
-#include <DPPP/H5Parm.h>
 #include <iostream>
 #include <sstream>
 #include <stdio.h>
 #include <vector>
-#include <Common/LofarLogger.h>
 #include <casacore/casa/BasicMath/Math.h>
 
+#include <boost/test/unit_test.hpp>
+
+#include "../../H5Parm.h"
+
 using namespace std;
-using namespace LOFAR;
+using namespace DP3;
+
+BOOST_AUTO_TEST_SUITE(h5parm)
 
 void checkAxes(H5Parm::SolTab& soltab, size_t ntimes) {
-  ASSERT(soltab.nAxes()==3);
-  ASSERT(soltab.hasAxis("ant"));
-  ASSERT(soltab.hasAxis("time"));
-  ASSERT(soltab.hasAxis("bla"));
-  ASSERT(soltab.getAxis(0).name=="ant");
-  ASSERT(soltab.getAxis(1).name=="time");
-  ASSERT(soltab.getAxis(2).name=="bla");
-  ASSERT(soltab.getAxis(0).size==3);
-  ASSERT(soltab.getAxis(1).size==ntimes);
-  ASSERT(soltab.getAxis(2).size==1);
+  BOOST_CHECK_EQUAL(soltab.nAxes(), size_t {3});
+  BOOST_CHECK(soltab.hasAxis("ant"));
+  BOOST_CHECK(soltab.hasAxis("time"));
+  BOOST_CHECK(soltab.hasAxis("bla"));
+  BOOST_CHECK_EQUAL(soltab.getAxis(0).name, "ant");
+  BOOST_CHECK_EQUAL(soltab.getAxis(1).name, "time");
+  BOOST_CHECK_EQUAL(soltab.getAxis(2).name, "bla");
+  BOOST_CHECK_EQUAL(soltab.getAxis(0).size, size_t {3});
+  BOOST_CHECK_EQUAL(soltab.getAxis(1).size, ntimes);
+  BOOST_CHECK_EQUAL(soltab.getAxis(2).size, size_t {1});
 }
 
-int main(int, char**) {
+BOOST_AUTO_TEST_CASE( test_gridinterpolate ) {
   {
     size_t ntimes=7;
     {
       // Create a new H5Parm
-      cout<<"Create tH5Parm_tmp.h5"<<endl;
+      // cout<<"Create tH5Parm_tmp.h5"<<endl;
```

Comment:
Most of these cout-statement can just be removed IMHO; maybe some can be changed into a comment to explain what's done